docs/landing-zone-v2/README.md

```diff
@@ -396,25 +396,33 @@ cd pbmm-landingzone
 
 1. Get the gatekeeper policies package. These are the policies which your Config Controller instance will use to [validate](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller) compliance of any configuration deployed, this currently checks for guardrails compliance and naming conventions on projects. These policies will also be validated when running the `kpt fn render` command for early feedback.
 
+   The latest versions of the releases can be found in [Releases](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/releases). The provided versions are examples and may not be up to date.
```

Comment:
here is the code snippet you can use to extract the latest version
URL="https://raw.githubusercontent.com/GoogleCloudPlatform/pubsec-declarative-toolkit/main/.release-please-manifest.json"
PACKAGE="solutions/client-setup"

VERSION=$(curl -s $URL | jq -r ".\"$PACKAGE\"")

Reply:
@cartyc let me know if you are considering implement the suggested code now or if you prefer to do it later.

Reply:
Added the changes yesterday and didn't have time to push the code. Incoming :)